### PR TITLE
Adding geoschem-gcpy

### PR DESCRIPTION
**Pull request by WilliamDowns**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by WilliamDowns** [comment]
@conda-forge-admin, please ping team

This PR is ready for review. Thank you for your time!

File: recipes/geoschem-gcpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+  doc_url: https://gcpy.readthedocs.io/en/latest/
+  dev_url: https://github.com/geoschem/gcpy
+
+extra:
+  recipe-maintainers:
+    - WilliamDowns
+    - yantosca
+    - lizziel
+    - msulprizio
+    - LiamBindle
```
Comment (ocefpaf) [inline recipes/geoschem-gcpy/meta.yaml:69]: @WilliamDowns we need all these folks to confirm that they are OK being maintainers of this recipe.

**Comment by WilliamDowns** [comment]
@yantosca, @lizziel, @msulprizio, @LiamBindle: is this okay with you?

**Comment by yantosca** [comment]
This is OK with me.

**Comment by LiamBindle** [comment]
Looks great!

**Comment by msulprizio** [comment]
Yes, it's OK with me.

**Comment by lizziel** [comment]
Fine with me!

File: recipes/geoschem-gcpy/meta.yaml (hunk 1 above)
Comment (WilliamDowns) [reply recipes/geoschem-gcpy/meta.yaml:69]: @ocefpaf All maintainers have now confirmed below.

**Pull request merged**